test/gtest/lppool.cpp

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+#include "lppool.hpp"
+#include "gtest/gtest.h"
+using float16 = half_float::half;
+
+// FORWARD TEST
+using GPU_LPPool_fwd_FP32  = LPPoolTestFwd<float>;
+using GPU_LPPool_fwd_FP16  = LPPoolTestFwd<float16>;
+using GPU_LPPool_fwd_BFP16 = LPPoolTestFwd<bfloat16>;
+
+TEST_P(GPU_LPPool_fwd_FP32, LPPoolTestFwd)
```

Comment:
```suggestion
TEST_P(GPU_LPPool_fwd_FP32, Test)
```
based on this naming convetion https://github.com/ROCm/MIOpen/wiki/GTest-development

Reply:
resolved